buildtools/npm/npm.go

```diff
@@ -54,3 +66,23 @@ func (n *NPM) Install(dir string) error {
 	}
 	return nil
 }
+
+func New(options map[string]interface{}) (NPM, error) {
```

Comment:
I don't think we actually use the `options` for anything, so feel free to get rid of them.